Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Build and run via:
+#   docker build -t codeclimate/codeclimate-reek . && docker run codeclimate/codeclimate-reek
+
+FROM codeclimate/alpine-ruby:b38
+
+MAINTAINER The Reek core team
+
+ENV code_dir /code
+ENV app_dir /usr/src/app
+ENV user app
+
+RUN apk --update add git
+
+ADD . ${app_dir}
+
+WORKDIR ${app_dir}
+
+RUN bundle install --without local_development development
```

Comment:
Yep, but then development would be more tedious. If you'd like to work on the docker set up including our latest changes (which **is** the default, at least that would make sense to me) you'd have to switch this line back and forth.
Additionally this seems to be idiomatic, at least if I check out the other codeclimate related docker files.